Dockerfile

```diff
@@ -1,14 +1,22 @@
-FROM python:3.8-slim
+FROM python:3.10-slim
 
 WORKDIR /app
 
 COPY . .
 
-ARG HEROKU
```

Comment:
Yeah it's set automatically only in Heroku docker deployment, but if it needs to be removed, I'd remove all parts of it instead of just one.

Reply:
no it don't has to be removed... we could also just set the default to 0 like:
`ENV HEROKU=0`
or leave it empty. But in some docker gu's it could confuse people who don't know these things, so i would at least set it to 0.

Reply:
Would `ENV HEROKU=$HEROKU` suffice? For me it's quite clear, "copy the value from outside to inside of the container".

Reply:
Ah so if theres a heroku db on the hostsystem it uses that? I have to admit that i don't do anything in this regard so i'm clueless in how it's getting used 😁